stories/va-accordion.stories.jsx

```diff
@@ -77,10 +76,10 @@ Default.args = {
 };
 Default.argTypes = propStructure(accordionDocs);
 
-export const Multi = Template.bind({});
-Multi.args = {
+export const NoMulti = Template.bind({});
+NoMulti.args = {
```

Comment:
```suggestion
export const OnlyOpenOne = Template.bind({});
OnlyOpenOne.args = {
```
I'm thinking of the sidebar nav menu. Seeing "No Multi" is maybe a bit awkward. "Open Only One" is maybe less awkward? I'm game to change it, but our goal is to make it obvious to a new engineer looking at Storybook for the first time to know what all the things do by the story name.